editor/src/components/canvas/ui-jsx-canvas.test-utils.tsx

```diff
@@ -155,8 +154,7 @@ export function renderCanvasReturnResultAndError(
   const baseRequireFn = getRequireFn(NO_OP, updatedContents, {}, {}, 'canvas')
   const requireFn: UiJsxCanvasProps['requireFn'] = (importOrigin: string, toImport: string) => {
     switch (toImport) {
-      case 'antd':
-        return ANTD
+      // here we can manually insert extra dependencies, such as antd
```

Comment:
I left this as a placeholder here, but I removed antd because I deleted the one test that used it